src/custom-components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import { usePagination, DOTS } from "./usePagination";
+
+export interface PaginationInterface {
+  totalCount: number;
+  pageSize: number;
+  currentPage: number;
+  onPageChange: (arg: number) => void;
+}
+
+const Pagination: React.FC<PaginationInterface> = ({
```

Comment:
There is a lot of logic in this component, it would be nice to have tests to cover it.

Reply:
our team decided to remove the pagination component from this scope. We will include it in the next PR with the corresponding tests